dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -338,6 +340,15 @@ public Contentlet findContentletByIdentifier(String identifier, boolean live, lo
 
     }
 
+    public List<Contentlet> findPagesByTemplate(Template template, User user, boolean respectFrontendRoles) throws DotDataException, DotSecurityException {
+        try {
+            return permissionAPI.filterCollection(search("+_all:" + template.getIdentifier() + " +baseType:" + BaseContentType.HTMLPAGE.getType(), -1, 0, null , user, respectFrontendRoles), PermissionAPI.PERMISSION_READ, respectFrontendRoles, user);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 249 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)